scribble-doc/scribblings/scribble/acmart.scrbl

```diff
@@ -35,9 +44,8 @@ Example:
 @defidform[sigplan]
 @defidform[sigchi]
 @defidform[sigchi-a]
-@defidform[dtrap]
```

Comment:
Would it be possible to preserve these so that old programs that use them can still be run?